server/controller/websocket.py

```diff
@@ -112,12 +111,16 @@ def log_dump(message, websocket, metadata):
     parsed_logs = LogParser.parse(message)
     api_key = metadata.get('apiKey', '')
 
-    # Send to database and convert to html.
-    html_logs = LogParser.convert_to_html(parsed_logs['logEntries'])
+    # Send to database.
     controller.datastore_interface.store_logs(
         api_key, metadata['deviceName'], metadata['appName'],
         metadata['startTime'], metadata['osType'], parsed_logs)
 
+    # Convert to html by creaing an array of all the converted rows.
+    html_logs = []
```

Comment:
this is simple enough to use a list comprehension:

    html_logs = [LogParser.convert_line_to_html(log) for log in parsed_logs['logEntries']]